src/core/book-keeper.js

```diff
@@ -1,9 +1,34 @@
 'use strict';
 
+var events = require('./event/event');
+var config = require('../../config/default');
+
 class BookKeeper {
-    constructor () {
+    constructor (eventBus) {
+
+        eventBus.emitEvent(events.FILL_REMINDER_QUEUE, { }, function () {
+
+        });
 
+        setInterval(function () {
+            eventBus.emitEvent(events.FETCH_NEXT_REMINDER, {}, function (err, data) {
```

Comment:
There needs to be a check for null/empty data (no reminders) which would be a NoOp